assets/src/edit-story/karma/fixture.js

```diff
@@ -230,6 +230,24 @@ export class Fixture {
     return this._container.querySelectorAll(selector);
   }
 
+  /**
+   * @param {Element} element
+   * @return {Promise} Yields when the element is displayed on the screen.
+   */
+  waitOnScreen(element) {
+    return new Promise((resolve) => {
+      const io = new IntersectionObserver((records) => {
+        records.forEach((record) => {
+          if (record.isIntersecting) {
+            resolve();
```

Comment:
Can this resolve twice (two records intersecting)? And would that be a problem?

Reply:
We can resolve many times. Only the first one will take. Several records intersecting or intersecting/non-intersecting in this case, I don't think that's important for this specific question.